dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/net/NetworkTimeInterceptor.kt

```diff
@@ -43,7 +43,9 @@ internal class NetworkTimeInterceptor(
 
         val serverDateStr = response.header("date").orEmpty()
         val serverDate = try {
-            formatter.parse(serverDateStr)
+            synchronized(this) {
```

Comment:
That's interesting

Reply:
I am thinking maybe we should synchronize on the formatter instead ...

Reply:
That's a thought